devicon.json

```
@@ -2354,20 +2354,26 @@
     {
         "name": "ember",
         "altnames": [
-            "emberjs"
+            "ember.js"
         ],
         "tags": [
-            "framework"
+            "framework",
+            "javascript",
+            "web-application",
```

Same as with the grails PR, it's not really a web-application, but it's used to make web-application, so another tag might be more fitting
```suggestion
            "web",
```

And/or
```suggestion
            "web-development"
```